### DISCUSSION TITLE
Show all possible state values in a dropdown for State Triggers/Conditions

### DISCUSSION THREAD

**Discussion by niemyjski** [home-assistant/frontend#7826] [Automations / scripts / scenes]
It would be awesome to be able to see all previously recorded state values (think terms aggregation) for an entity or entity state attribute (Even a dropdown for attributes too!).  This would take editing automations to a whole new level and you could even add a strict validation mode to know if you are possibly writing an automation that will never work.

One could then use a dropdown to able to select this in the `From`, `To` or `State` values or manually provide a value which has yet to be seen. We could even add in known values for the sensor types/device classes we know about.

**Comment by KTibow** [comment]
This might have already been discussed.

**Reply by niemyjski** [reply]
I didn't see anything in the discussions when I tried searching for it. But I could have been missing a search term and didn't see it.

**Reply by KTibow** [reply]
I couldn't find it either. Sorry about this.
